.github/workflows/build_and_test_debug.yml

```diff
@@ -133,7 +133,9 @@ jobs:
           go-version: 1.18
 
       - name: Install Go Binary
-        run: go install golang.org/x/mobile/cmd/gomobile
+        run: |
+          go install golang.org/x/mobile/cmd/gomobile
```

Comment:
Let's make sure we use gomobile consistently. This is how we are using in the SDK: https://github.com/Jigsaw-Code/outline-sdk/tree/main/x/mobileproxy

Notes:
- gomobile init is not needed
- gobind must be in the PATH
- prefer go build to go install because it's more explicit and easier to understand

/cc @jyyi1